python/src/graph_window.rs

```diff
@@ -168,31 +173,66 @@ impl WindowedVertex {
 #[pymethods]
 impl WindowedVertex {
 
-    pub fn __getitem__(&self, name: String) -> Vec<(i64, Prop)> {
-        self.prop(name)
+    pub fn __getitem__(&self, name: String) -> Option<Prop> {
+        self.property(name,Some(true))
+    }
+
+    pub fn has_property(&self,name:String,include_static:Option<bool>) -> bool {
+        match include_static {
```

Comment:
in all these functions, instead of pattern matching `include_static` we could do:
```rust
let include_static = include_static.unwrap_or(true)
self.vertex_w.has_property(name,include_static)
```